integration-tests/deployment/ccip/propose.go

```diff
@@ -217,3 +217,40 @@ func GenerateAcceptOwnershipProposal(
 		batches,
 		timelock.Schedule, "0s")
 }
+
+func CreateSingleChainMCMSOps(
```

Comment:
@0xAustinWang this is taken from your PR, just added the `description` param

Reply:
should rename as this is actually building a proposal